src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +338,43 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    static String[] getDockerGroupCmdArgs(String scanReportPath) {
+        String[] cmdGroupArgs = {"", ""};
+        if (!scanResultsFileExist(scanReportPath)) {
+            cmdGroupArgs[0] = "-u";
+            cmdGroupArgs[1] = executeCommand("id -g");
```

Comment:
For me `id -g` does not work.. It makes existing tests fail. This works for me when the value is `id -u`..

Reply:
I changed by `id - u` for me worked as well.